rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -106,10 +109,10 @@ def _validate_model_files_exist(model_directory: Text) -> None:
         for file_path in files_to_check:
             if not os.path.exists(file_path):
                 raise RasaException(
-                    f"""File {file_path} does not exist.
-                        Re-check the files inside the directory {model_directory}.
-                        It should contain the following model
-                        files - [{", ".join(files_to_check)}]"""
+                    f"File {file_path} does not exist. "
+                    f"Re-check the files inside the directory {model_directory}. "
+                    f"It should contain the following model "
+                    f'files - [{", ".join(files_to_check)}]'
```

Comment:
Optional: I think this is more consistent with the rest of the codebase
```suggestion
                    f"files - [{', '.join(files_to_check)}]"
```